src/XeroPHP/Remote/Query.php

```diff
@@ -105,16 +161,46 @@ public function offset($offset = 0) {
     }
 
     /**
+     * @param bool $includeArchived
+     * @return $this
+     */
+    public function includeArchived($includeArchived = true) {
```

Comment:
@calcinai the reason for parameter here is that you can includeArchived by just using it, or you can disable the condition by calling includeArchived(false);

``` php
// Prepare your query in some method
$query->includeArchived(); // archived are going to be included

// In other method I get prepared query
if ($something === true) {
    $query ->includeArchived(false); // will not include archived anymore.
}
```

I don't need this functionality for myself. So we can remove it for now and make it argument less, but I think for anyone who would need this it might be a nice feature.